### DISCUSSION TITLE
for KNN decoder

### DISCUSSION THREAD

**Discussion by yuhangzhu4** [AdaptiveMotorControlLab/CEBRA#7] [General]
Can I save the KNN decoder after training the decoder on the training set?I want to load the file when predicting new data

**Comment by MMathisLab** [comment]
hi @yuhangzhu4 - yes, the knn decode is from sklearn, so please see their docs, and then its straightforward to write code to save hte outputs: https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html
